src/GitHash.hs

```diff
@@ -111,6 +113,10 @@ giCommitCount = _giCommitCount
 giCommitMessage :: GitInfo -> String
 giCommitMessage = _giCommitMessage
 
+-- @since 0.1.2.0
```

Comment:
This should include a full Haddock comment like the other identifiers, as well as the new version number, which should be 0.1.4.0. This change should also include a changelog entry and version bump.